api/src/shared/infrastructure/datasources/pix-assets/pix-assets-client.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * @param{URL} assetUrl
+ */
+export async function fetchAssetsMetadata(assetUrl) {
+  const assetResponse = await fetch(assetUrl, { method: 'OPTIONS' });
```

Comment:
Est-ce que
```suggestion
  const assetResponse = await fetch(assetUrl, { method: 'HEAD' });
```
ne serait pas plus approprié ?

`OPTIONS` ça semble être spécifiquement pour récupérer les méthodes HTTP supportés par un endpoint.

Reply:
alors j'ai essayé ça d'abord mais sans réussite... en l'état ça semble fonctionner 🙈